### DISCUSSION TITLE
Welcome to cockpit-files Discussions!

### DISCUSSION THREAD

**Discussion by martinpitt** [cockpit-project/cockpit-files#209] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by B3rn475** [comment]
I guess the title of announcement should be updated. 😉

Thank You for the great work.

**Reply by martinpitt** [reply]
Right, thanks for pointing out! Fixed.
